page_objects/registration-page.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+const LoginPage = require("../page_objects/login-page")
+const faker = require("faker");
+const mailosaurTest = require("../data/mailosaurTest.json");
+const MailosaurClient = require("mailosaur");
+const randomEmail =
+  faker.internet.userName() + mailosaurTest.register.serverDomain;
+const randomPassword = faker.internet.password();
+const randomUsername = faker.internet.userName();
+
+class Registration {
+  get $sportsCommunityLbl() {return $("span=Sports");}
+  get $creatorsToFollowDisplay() {return $('[class="creators-wrapper mb-3"]');}
+  get $startExploringBtn() {return $("button=Start exploring");}
+  get $continueBtn() {return $("button=Continue");}
+  get $welcomeBackLbl() { return $("h3=Welcome back to the community"); }
+  get $joinTodayBtn() { return $(".ml-2"); }
+
+  // Registration
+  register(portal) {
+    if (portal === "brands") {
+      // Click on Brands register radio button
+      LoginPage.$brandsLnk.click();
+      // Type in email and password
+      LoginPage.$emailTxt.waitForClickable();
+      LoginPage.$emailTxt.setValue(randomEmail);
+      LoginPage.$passwordTxt.setValue(randomPassword);
+    } else if (portal === "creators") {
+      LoginPage.$passwordTxt.waitForClickable();
+      LoginPage.$emailTxt.setValue(randomEmail);
+      LoginPage.$passwordTxt.setValue(randomPassword);
+      LoginPage.$usernameTxt.setValue(randomUsername);
+      LoginPage.$continueLnk.click();
+    }
```

Comment:
Add else and throw error in there, in case someone will not specify brands or portals